### ISSUE TITLE
Android P doesnt display toast

### ISSUE THREAD

**Issue by northkode** [EddyVerbruggen/Toast-PhoneGap-Plugin#119]
### Expected behaviour
Toast to show

### Actual behaviour
Android P does not show the toast over the webview.

### I'm seeing this behaviour on
- [ ] iOS device
- [ ] iOS sim
- [x] Android device (only one I am using)
- [ ] Android sim

#### Hardware models
Pixel 2 XL

#### OS versions
Android P

### I'm not a dummy, so I've checked these
- [x] It happens on a fresh Cordova CLI project as well.
- [x] I'm waiting for `deviceready` to fire.
- [x] My JavaScript has no errors (`window.onerror` catches nothing).

### So how can we reproduce this?
The Toast success is called and I assume the toast code runs in the background but it would appear that for some reason its not above the webview.

**Comment by DavidWiesner** [comment]
I have the same issue with android pie. In the log shows the following messages that mabe related to this issue:
```
09-14 07:27:00.397 4193-4745/? I/NotificationService: cancelToast pkg=com.blah.app callback=android.app.ITransientNotification$Stub$Proxy@118871c
09-14 07:27:00.397 4193-4745/? W/NotificationService: Toast already cancelled. pkg=com.com.blah.app callback=android.app.ITransientNotification$Stub$Proxy@118871c
```

**Comment by EddyVerbruggen** [comment]
Likely fixed with 2.7.1.
